pkg/cmd/issue/view/issuePrintFormatter.go

```diff
@@ -0,0 +1,132 @@
+package view
```

Comment:
**Requested Change**

It's conventional in Go (but more importantly, in this codebase) to use underscores in file names so please make that change here and the test file.

Reply:
Ah, right. Thanks

Reply:
I think I've done this. Let me know if I missed any